### PR TITLE
zotero extension for searching zotero database directly from raycast

### PR DESCRIPTION
**Pull request by sadanand-singh**
## Description

A simple extension to search your zotero database directly from Raycast

## Type of change

- New extension

## Checklist

- [x ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

File: extensions/zotero/package.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
+  "name": "zotero",
+  "title": "Search Zotero",
+  "description": "Search Zotero Databae from Raycast",
```
Comment (fe9lix) [inline extensions/zotero/package.json:4]: Typo: Databae

File: extensions/zotero/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "Node 16",
+  "include": ["src/**/*"],
+  "compilerOptions": {
+    "lib": ["es2020"],
+    "module": "commonjs",
+    "target": "es2020",
+    // "strict": true,
```
Comment (fe9lix) [inline extensions/zotero/tsconfig.json:9]: Is there a specific reason you want to disable this?

**Review by fe9lix — CHANGES_REQUESTED** [review]
Hey @sadanand-singh , thanks for your contribution!

Is the extension still WIP? If so, you might want to set this PR to a "Draft" and then request a review once you're ready.

Unfortunately I can't fully test the extension since I run into a JSON parsing error when loading the list: 

```
runQuery error SyntaxError: Unexpected token A in JSON at position 0
    at JSON.parse (<anonymous>)
```
    
Can you double check? Chances are other users might run into the same issue (not sure if it has something to do with the fact that I've created a new account and test token...)

I've also left some inline comments.

**Comment by fe9lix** [comment]
> Here is a test account:
> 
> key: `MnnQ8wKPuu2noKmnJk48PCoH` user id: `8623690`

Thanks! Please make sure to disable that test account again after the initial review since the credentials are public here.

**Comment by gumadeiras** [comment]
would it make sense to search a local zotero database (json or bibtex) as an alternative to the API? that way you don't need internet access and searching should be much faster

**Pull request merged**